homeassistant/components/squeezebox/media_player.py

```diff
@@ -214,11 +185,18 @@ class SqueezeBoxEntity(MediaPlayerEntity):
     _last_update: datetime | None = None
     _attr_available = True
 
-    def __init__(self, player: Player, server: Server) -> None:
+    def __init__(
+        self,
+        coordinator: SqueezeBoxPlayerUpdateCoordinator,
+    ) -> None:
         """Initialize the SqueezeBox device."""
+        super().__init__(coordinator)
+        self._coordinator = coordinator
```

Comment:
It's already stored at `self.coordinator`

Reply:
Ah, got it. Thank you.